scripts/s3/s3api/bucket_objects_operations.py

```diff
@@ -60,23 +60,31 @@ async def execute_bucket_object_workload(self):
             number_of_objects = self.kwargs.get("number_of_objects")
             delete_percentage_per_bucket = self.kwargs.get("delete_percentage_per_bucket")
             put_percentage_per_bucket = self.kwargs.get("put_percentage_per_bucket")
+            overwrite_percentage_per_bucket = self.kwargs.get("overwrite_percentage_per_bucket")
```

Comment:
I can see overwrite distribution is not calculated. For same refer line no 70 and make it conditional.

Reply:
Done